CHANGES.rst

```diff
@@ -3,6 +3,25 @@
 CHANGES
 =======
 
+
+API
```

Comment:
Usually API refers the external API - things that a person using core would have to worry about. 

I suspect this is just about internal organization and under the general category of we are moving evaluation functions into its own module.